LuaMenu/widgets/snd_music_lite.lua

```diff
@@ -26,6 +26,23 @@ local randomTrackList = {
 	"sounds/lobbyMusic/Interstellar.ogg",
 	"sounds/lobbyMusic/Tomorrow Landscape.ogg",
 }
+local OPEN_TRACK_NAME = 'sounds/lobbyMusic/The Secret of Ayers Rock.ogg'
+
+-- load custom game dependent music
+if WG.Chobby then
```

Comment:
This should all happen in initialize (or delayedInitialize) as checking WG while file loading is just bad. Also, instead of keeping the hardcoded track name in addition to an optional override the track configuration should be ported into game config.

Reply:
Indeed, any chance for a quick fix @Ruwetuin?

Reply:
done in PR, and removed the default zk tracks hardcoded in the widget